mod.conf

```diff
@@ -1,6 +1,5 @@
 name = goodtorch
 title = Good Torch
 author = LissoBone
-depends = default
-optional_depends = technic
+optional_depends = technic, default, mcl_core, mcl_sounds
```

Comment:
Moved from a single hard dependency to multiple soft dependencies (with code further above `error` that will abort if the user doesn't have one or another of these soft depends)